migrations/1655893804425-init.ts

```diff
@@ -0,0 +1,63 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class init1655893804425 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
```

Comment:
Shouldn't we check if table exists before we create? Or is that already an existing behaviour of the `queryRunner`?

Reply:
Typeorm handles that for us - if it sees that particular migration has been applied it won't run this query

Reply:
So, those who have initially deployed the volume has to clear their files in order to properly run this migration instance?